Branch-SDK/src/main/java/io/branch/referral/ServerRequestCreateUrl.java

```diff
@@ -122,7 +122,13 @@ public boolean handleErrors(Context context) {
     @Override
     public void onRequestSucceeded(ServerResponse resp, Branch branch) {
         try {
-            final String url = resp.getObject().getString("url");
+            String url = resp.getObject().getString("url");
+
+            AttributionReportingManager attributionManager = AttributionReportingManager.INSTANCE;
+            if (attributionManager.isMeasurementApiEnabled()) {
+                url = url + "&" + Defines.Jsonkey.Privacy_Sandbox.getKey() + "=true";
```

Comment:
Is this part of the product spec? To include this url param, confirm with Product

Reply:
Just confirmed this